start-from-zero/Javascript/src/views/new-evaluation/new-evaluation.js

```diff
@@ -0,0 +1,725 @@
+                    <input class='input-calendar' type="date">
+                </div>
+
+
+                <h4 class='technical-header'>Technical Level</h4>
+                <table class='technical-level-table'>
+
+                    <tbody class='tbody-new'>
+                        <tr class='technical-level'>
+                            ${TechnicalLevel({
+                                el: options.el
+                            })}
+                        </tr>
+
+
+                        <tr class="technical-level-category">
+                            <td>
+                                <input class='input-checkbox' type="checkbox" name='trainee'>
+                            </td>
+                            <td>
+                                <input class='input-checkbox' type="checkbox" name='junior1'>
+                                <input class='input-checkbox' type="checkbox" name='junior2'>
+                                <input class='input-checkbox' type="checkbox" name='junior3'>
+                            </td>
+                            <td>
+                                <input class='input-checkbox' type="checkbox" name='middle1'>
+                                <input class='input-checkbox' type="checkbox" name='middle2'>
+                                <input class='input-checkbox' type="checkbox" name='middle3'>
+                            </td>
+                            <td>
+                                <input class='input-checkbox' type="checkbox" name='senior1'>
+                                <input class='input-checkbox' type="checkbox" name='senior2'>
+                                <input class='input-checkbox' type="checkbox" name='senior3'>
+                            </td>
+                        </tr>
+                    </tbody>
+                </table>
+
+
+
+
+                <div class="textarea-wrap">
+                    <h4 class='textarea-header'>Should the candidate be hired?</h4>
+                    <textarea class='textarea' name="" id="" rows="10" placeholder="The type of project that is suitable for the candidate Is guidance required for the candidate"></textarea>
+                </div>
+
+                <div class="textarea-wrap">
+                    <h4 class='textarea-header'>General impression</h4>
+                    <textarea class='textarea' name="" id="" rows="10" placeholder="The type of project that is suitable for the candidate Is guidance required for the candidate" required></textarea>
+                </div>
+
+                <div class="textarea-wrap">
+                    <h4 class='textarea-header'>Workflow, Leadership and Soft Skills</h4>
+                    <textarea class='textarea' name="" id="" rows="10" placeholder="Describe the environment in which the candidate works. Describe any guidance or management experience."></textarea>
+                </div>
+
+
+
+
+                <fieldset class='fieldset'>
```

Comment:
create a component for each fieldset. supply it's data from an array of objects

Reply:
Solved. Thanks Gabi for your guidance and advice. :)